test/functional/readconcern_tests.js

```diff
@@ -435,7 +435,7 @@ describe('ReadConcern', function() {
   });
 
   it('Should set majority readConcern aggregate command but ignore due to out', {
-    metadata: { requires: { topology: 'replicaset', mongodb: '>= 3.2' } },
+    metadata: { requires: { topology: 'replicaset', mongodb: '>= 3.2 < 4.1' } },
```

Comment:
I didn't realize we could do this!

Reply:
yeah, [the underlying check we use](https://github.com/mongodb-js/mongodb-test-runner/blob/master/lib/filters/mongodb_version_filter.js#L77) leverages [semver](https://www.npmjs.com/package/semver), so any valid semver expression will work.